spec/models/concerns/offer_search_spec.rb

```diff
@@ -1,76 +1,76 @@
 require 'rails_helper'
 
 context OfferSearch do
-  
+
   let!(:offer) { create :offer, :submitted } # exists to confirm we never find it.
 
   # integration tests are located in offers_controller_spec for "GET offers/search"
   context "searches for" do
 
     context "user first_name" do
       let!(:offer1) { create :offer, :submitted, created_by: (create :user, first_name: 'Test') }
-      it { expect(Offer.search(search_text: 'Test').to_a).to match_array([offer1]) }
+      it { expect(Offer.search({search_text: 'Test', :states=>[]}).to_a).to match_array([offer1]) }
```

Comment:
I have added it here because in controller where I am creating `hash` for `search` method it is always going to send `states` array even if it is empty.

Reply:
👍